its/src/test/java/com/sonar/it/scanner/msbuild/utils/TestUtils.java

```diff
@@ -97,16 +97,21 @@ public static String developmentScannerVersion() {
   }
 
   // ToDo: SCAN4NET-201: Remove Orchestrator
-  public static ScannerCommand newScanner(Orchestrator orchestrator, Path projectDir, String token) {
+  public static ScannerCommand newScannerBegin(Orchestrator orchestrator, String projectKey, Path projectDir, String token) {
```

Comment:
I think the `new` prefix redundant. This is nothing that should be fixed a part of this PR.

Reply:
The `ToDo` says to remove it, so I'll leave it for now :)